webrtc/videoroom/lib/videoroom/room.ex

```diff
@@ -53,7 +53,10 @@ defmodule Videoroom.Room do
 
   @impl true
   def handle_info({_sfu_engine, {:sfu_media_event, to, event}}, state) do
-    send(state.peer_channels[to], {:media_event, event})
+    if state.peer_channels[to] != nil do
```

Comment:
Can this ever happen? We always call `add_peer_channel` as soon as new WS is created so it shouldn't be possible to trigger WS to emit message to the peer before calling `add_peer_channel`.

Reply:
It is possible in situation when two peers left room in same time and then the message peerLeft is sent to peer which already left.